tools/block-generator/README.md

```diff
@@ -133,18 +133,21 @@ Usage:
 
 Flags:
   -i, --conduit-binary string               Path to conduit binary.
-      --cpuprofile string                   Path where conduit writes its CPU profile.
+  -l, --conduit-log-level string            LogLevel to use when starting Conduit. [panic, fatal, error, warn, info, debug, trace] (default "error")
```

Comment:
With the introduction of `--verbose` for printing out generator/ledger info, I want to disambiguate that the log level belongs to `conduit` so changing `--log-level` &rarr; `--conduit-log-level`